src/main/java/com/gooddata/dataload/processes/ScheduleExecution.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (C) 2004-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.dataload.processes;
+
+import static com.gooddata.util.Validate.notNullState;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.gooddata.util.ISODateTimeDeserializer;
+import org.joda.time.DateTime;
+import org.springframework.web.util.UriTemplate;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Schedule execution
+ */
+@JsonTypeInfo(include = JsonTypeInfo.As.WRAPPER_OBJECT, use = JsonTypeInfo.Id.NAME)
+@JsonTypeName("execution")
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ScheduleExecution {
+
+    public static final String URI = "/gdc/projects/{projectId}/schedules/{scheduleId}/executions/{executionId}";
+    public static final UriTemplate TEMPLATE = new UriTemplate(URI);
+    public static final List<String> FINISHED_STATUSES = Arrays.asList("OK", "ERROR", "CANCELED", "TIMEOUT");
```

Comment:
@martiner https://github.com/gooddata/msf-dto/blob/master/src/main/java/com/gooddata/service/model/scheduler/ExecutionStatus.java#L13

So it's IMHO `CANCELED`